spec/features/stories_spec.rb

```diff
@@ -39,6 +39,49 @@
       end
     end
 
+    context 'when the user save a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'highlights story on click to save', js: true do
+        find(story_selector(story)).trigger('click')
+        expect(page).to have_css('.editing')
+        click_on 'Save'
+        expect(page).not_to have_css('.editing')
+        expect(find(story_selector(story))[:style]).to match(/background-color/)
+      end
+    end
+
+    context 'when the user drags a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'check if the story is in the correct column', js: true do
+        expect(find('#in_progress')).to have_css("div#{story_selector(story)}")
+        expect(find('#chilly_bin')).not_to have_css("div#{story_selector(story)}")
+      end
+
+
+      it 'drags a story to other column', js: true do
+        source = page.find(story_selector(story))
+        target = page.find('#chilly_bin')
+        source.drag_to(target)
+
+        expect(find('#chilly_bin')).to have_css("div#{story_selector(story)}")
```

Comment:
Can we add some more cases where we'd try to drag clicking over some of the buttons of the story element? And another one when trying to drag clicking over the story description text (to see if it's actually gonna drag it or just select the text)? We should stress the cases where the drag should/shouldn't work to avoid regression when adding new features.